dist/chart/templates/manager/manager.yaml

```diff
@@ -44,7 +44,7 @@ spec:
             {{- end}}
           command:
             - /manager
-          image: {{ .Values.controllerManager.container.image.repository }}:{{ .Values.controllerManager.container.image.tag }}
+          image: {{ .Values.controllerManager.container.image.containerRegistry }}/{{ .Values.controllerManager.container.image.repository }}:{{ .Values.controllerManager.container.image.tag }}
```

Comment:
This doesn't look right, the `dist/chart` dir is generated.
You need to make changes in `config`.

Also, this looks like a breaking change, why can't the `repository` contain the full path w/ `/` ?